### PR TITLE
* supporting separation of read/write host data

### PR DESCRIPTION
**Pull request by erichulser**
* pooling based on host

### CODE REVIEW

File: orb/core/connection_types/sql/sqlconnection.py [hunk 1]
```diff
@@ -164,15 +164,16 @@ def close(self):
...
-                self._close(conn)
-            except Exception:
-                pass
+        for pool in self.__pool.values():
+            while not pool.empty():
+                conn = pool.get_nowait()
+                try:
+                    self._close(conn)
+                except Exception:
+                    pass
```
Comment (danielmurray) [inline orb/core/connection_types/sql/sqlconnection.py]: :/

File: orb/settings.py [hunk 2]
```diff
@@ -11,7 +11,7 @@ class Settings(object):
         'raise_background_errors': 'True',
         'caching_enabled': 'False',
         'max_cache_timeout': str(1000 * 60 * 60 * 24), # 24 hours
-        'max_connections': '3',
+        'max_connections': '10',
```
Comment (danielmurray) [inline orb/settings.py]: Are these default settings?

File: tests/postgres/test_pg_api.py [hunk 3]
```diff
@@ -407,4 +407,54 @@ def test_pg_sub_selection(orb, GroupUser, User):
...
+
+    # test read
+    new_open.read = True
+    comment = Comment.select().first()
+    assert comment is not None
+
+    # test update
+    new_open.read = False
+    comment.set('text', 'some new text')
+    comment.save()
```
Comment (danielmurray) [inline tests/postgres/test_pg_api.py]: assertion?

File: orb/settings.py (hunk 2 above)
Comment (erichulser) [inline orb/settings.py]: yup

File: tests/postgres/test_pg_api.py (hunk 3 above)
Comment (erichulser) [inline tests/postgres/test_pg_api.py]: assertion happens in the function above it

File: orb/core/connection_types/sql/sqlconnection.py (hunk 1 above)
Comment (erichulser) [inline orb/core/connection_types/sql/sqlconnection.py]: voodoo.

**Pull request merged**